ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/inmemory/InMemoryLedger.scala

```diff
@@ -273,4 +274,11 @@ class InMemoryLedger(
         }
       )
   }
+
+  override def publishConfiguration(
+      maxRecordTime: Time.Timestamp,
+      submissionId: String,
+      config: Configuration): Future[SubmissionResult] = ???
```

Comment:
Is it intentional to leave it unimplemented for in-mem sandbox?
Same comment applies to the statement 2 lines below